pkg/ethereum/nats.go

```diff
@@ -77,6 +128,21 @@ func (d *DiscoveryV5) sendPeerEvent(ctx context.Context, node *enode.Node, hInfo
 		Timestamp:  time.Now().UnixMilli(),
 	}
 
+	d.log.Info().
+		Str("enr", node.String()).
+		Str("id", hInfo.ID.String()).
+		Str("ip", hInfo.IP).
+		Int("port", hInfo.Port).
+		Str("crawler_id", peerEvent.CrawlerID).
+		Str("crawler_location", peerEvent.CrawlerLoc).
+		Msg("peer_discovered event")
+
+	if d.js == nil {
+		fmt.Fprintf(d.fileLogger, "%s ENR: %s, ID: %s, IP: %s, Port: %d, CrawlerID: %s, CrawlerLoc: %s\n",
```

Comment:
Let's print the stringified json of the event for consistency: `fmt.Fprintln(n.fileLogger, string(json))`